mage_ai/frontend/pages/pipelines/index.tsx

```diff
@@ -661,7 +657,15 @@ function PipelineListPage() {
   ]);
 
   const { data: dataTags } = api.tags.list();
-  const tags: TagType[] = useMemo(() => sortByKey(dataTags?.tags || [], ({ uuid }) => uuid), [
+  const tags: TagType[] = useMemo(() => {
+    const t = dataTags?.tags
+    if (t?.length > 0 && t[0].uuid !== 'no_tag') { // <- So as to not to re-add the `no_tag` tag.
```

Comment:
This seems to cause some unintended duplicates for the `no_tag` uuid.
![image](https://github.com/mage-ai/mage-ai/assets/78053898/0ec74db6-6349-41f4-8135-e51b44dd48ad)

Reply:
Ah, I was short-sighted - I didn't test the feature in every ways. I also saw you pushing, so it'd be lovely if I could take another issue to make it up.